flow/project.py

```diff
@@ -2922,10 +2960,24 @@ def _has_any_eligible_group(status_entry):
                 1 for _ in filter(_has_any_eligible_group, status_results)
             )
 
-        statuses = {
-            status_entry["aggregate_id"]: status_entry
-            for status_entry in status_results
-        }
+        def display_group_name(group_name):
+            """Return the operation name or group name with number of operations."""
+            # If the name is from a group that is not an operation, we append the
+            # number of operations to its name in the status.
+            if group_name not in self._operations:
+                num_operations = len(self._groups[group_name].operations)
+                return f"{group_name} ({num_operations} ops)"
+            return group_name
+
+        statuses = {}
+        # We store the name for display in statuses[aggregate_id][group_name][display_name] to
```

Comment:
The last key is not a variable.
```suggestion
        # We store the name for display in statuses[aggregate_id][group_name]["display_name"] to
```